etc/enforcer-rules/pom.xml

```diff
@@ -0,0 +1,191 @@
+                <groupId>org.eclipse.sisu</groupId>
+                <artifactId>sisu-maven-plugin</artifactId>
+                <version>0.9.0.M4</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>main-index</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>3.5.4</version>
+                <configuration>
+                    <skipTests>${skipUTs}</skipTests>
+                    <failIfNoTests>false</failIfNoTests>
+                    <runOrder>random</runOrder>
+                    <statelessTestsetReporter
+                            implementation="org.apache.maven.plugin.surefire.extensions.junit5.JUnit5Xml30StatelessReporter">
+                        <usePhrasedTestCaseMethodName>true</usePhrasedTestCaseMethodName>
+                    </statelessTestsetReporter>
+                    <argLine>
+                        @{jacoco.argline}
+                    </argLine>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-invoker-plugin</artifactId>
+                <version>3.2.2</version>
+                <executions>
+                    <execution>
+                        <id>integration-test</id>
+                        <configuration>
+                            <skipInstallation>${skipITs}</skipInstallation>
+                            <skipInvocation>${skipITs}</skipInvocation>
+                        </configuration>
+                        <goals>
+                            <goal>install</goal>
+                            <goal>run</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+    <profiles>
+        <profile>
+            <!-- note CI is deliberately defined after qa so that it can replace the formatter config plugin -->
+            <!-- see https://maven.apache.org/guides/introduction/introduction-to-profiles.html#profile-order -->
+            <id>ci</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <properties>
+                <sonar.organization>kroxylicious</sonar.organization>
+                <sonar.host.url>https://sonarcloud.io</sonar.host.url>
+                <impsort-maven-plugin.goal>check</impsort-maven-plugin.goal>
+                <skip-validate-yaml-snippets>false</skip-validate-yaml-snippets>
+            </properties>
+            <build>
+                <plugins>
+                    <plugin>
```

Comment:
Let's add https://central.sonatype.org/publish/publish-portal-maven/#skippublishing true.

Reply:
to prevent it publishing in central? I wonder if we do want to publish it though, to avoid our poms referencing anything that doesn't exist.